Dockerfile

```diff
@@ -1,25 +1,16 @@
-FROM python:3.7-alpine3.10 as builder
+FROM python:3.8-alpine as builder
 
 RUN apk add --no-cache \
     linux-headers \
     tcpdump \
     build-base \
-    ebtables
+    ebtables \
+    make \
+    git && \
+    apk upgrade --no-cache
 
 WORKDIR /kube-hunter
-COPY ./requirements.txt /kube-hunter/.
```

Comment:
Hm, it would be nice to have `pip install` layer cached, otherwise any code change triggers downloading the packages again, so rebuilding image takes ~30-40 sec.

Reply:
Good point.
Since requirements file doesn't include dependencies directly, we need another way of installing them and preserve check.
I can use `setup.py` custom command to list dependencies from `setup.cfg` and use it in Dockerfile

Reply:
Done